### ISSUE TITLE
Running as root without --no-sandbox is not supported

### ISSUE THREAD

**Issue by ZhengLin-Li** [saltyshiomix/nextron#309]
I run into an issue when start my first nextron app. It says 

```
[0111/120125.404317:FATAL:electron_main_delegate.cc(299)] Running as root without --no-sandbox is not supported
```

My envrionment:
node v19.4.0
npm 9.2.0
ubuntu 20.04

Commands I runned:
```bash
npx create-nextron-app my-app --example basic-typescript
npm install
npm run dev
```

The output log containing the problem:
```bash
root@DESKTOP-5J0RSVH ~/projects/Coding_In_University/Nextron/my-app
main* # npm run dev                                       [12:01:16]

> my-nextron-app@1.0.0 dev
> nextron

ready - started server on 0.0.0.0:8888, url: http://localhost:8888
event - compiled client and server successfully in 438 ms (158 modules)
[0111/120125.404317:FATAL:electron_main_delegate.cc(299)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.
/root/projects/Coding_In_University/Nextron/my-app/node_modules/electron/dist/electron exited with signal SIGTRAP
```

**Comment by saltyshiomix** [comment]
@ZhengLin-Li 

Thank you for reporting the issue!

We need to add the feature to pass options to electron.
I'll try it this or next week.

**Comment by OlekAleksander** [comment]
Same issue, any ETA?

**Comment by OlekAleksander** [comment]
Or a temporary fix?

**Comment by 22chrs** [comment]
any news here? :)

**Comment by badlektor** [comment]
I am not sure about the root part, but as a workaround for the sandbox part, until there is a fix, and in case someone is stuck, it is enough to locate 

`const w=()=>{t=n("electron",[".",`${y}`,`--remote-debugging-port=${g}`,`

on the file `nextron-dev.js` on your node_modules/nextron/bin folder

and add the --no-sandbox argument, like this 

`const w=()=>{t=n("electron",["--no-sandbox",".",`${y}`,`--remote-debugging-port=${g}`,`

**Comment by saltyshiomix** [comment]
@ZhengLin-Li CC: @OlekAleksander @22chrs @badlektor 

Sorry for late reply 🙇 

I just released nextron v8.11.1, which includes electron option configuration.
If you have time, please try new one and let me know how is it going 🙏 

https://github.com/saltyshiomix/nextron#--electron-options-default-undefined

```json
{
  "scripts": {
    "dev": "nextron --electron-options=\"--no-sandbox\""
  }
}
```

An example here: https://github.com/saltyshiomix/nextron/tree/v8.11.1/examples/basic-launch-app-from-url
